src/initializers/prometheus/index.ts

```diff
@@ -0,0 +1,22 @@
+import { getLogger } from '../log4js';
+import { snakeCase } from 'lodash';
+let instance;
```

Comment:
🙃  can we add typings in this instance? So that getPrometheus returns sth with types? There are available type only imports see: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html